cypress/support/component.tsx

```diff
@@ -0,0 +1,27 @@
+// ***********************************************************
+// This example support/component.ts is processed and
```

Comment:
Perhaps remove the word `example`? It's no longer an example, but something that we actually use?